src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -127,13 +128,20 @@ object EasyStageDataset {
           format = Some(mime),
           sha1 = maybeSha1Map.get.get(bagRelativePath), // first get is checked in advance
           title = title,
-          accessibleTo = FileAccessRights.accessibleTo(rights),
-          visibleTo = FileAccessRights.visibleTo(rights)
+          accessibleTo = FileAccessRights.accessibleTo(fileRights),
+          visibleTo = FileAccessRights.visibleTo(fileRights)
         )
         _ <- EasyStageFileItem.createFileSdo(sdoDir, "objectSDO" -> parentSDO)(fis)
       } yield ()
     }
 
+    def getFileRights(filePath: String)(implicit s: Settings): Try[AccessCategory] = {
+      readAccessRights(filePath) map {
+        case Some(fileRightsStr) => AccessCategory.valueOf(fileRightsStr)
```

Comment:
this is incorrect. if you have a `fileRightsStr` which comes from `files.xml \\ files \\ file \\ accessRight` then the value is something of a file access category; not of a dataset access category.
This is because of the changing requirements.

Also @lindareijnhoudt and I looked at how to calculate `accessibleTo` and `visibleTo`. We'll get back on that shortly...